### DISCUSSION TITLE
Default user / credentials ?

### DISCUSSION THREAD

**Discussion by Cloud-Kid** [tiredofit/docker-fusiondirectory#55] [Q&A]
Hi all,
First thanks for your containerizing job it looks really clean !

I have setup a test podman server to run fusion-directory which is very likely a product we really need.

I have setup the default docker-compose file with my own config for the ldap1 server and can reach the default web page. If I execute the --check-config it returns an OK status.

But I can't find where to specify the default credentials, are they LDAP credentials or local ?

**Comment by tiredofit** [comment]
You would login as fd-admin on the web interface for the first time with the password you setup with the `tiredofit/openldap-fusiondirectory` container.

**Reply by Cloud-Kid** [reply]
Thanks for your quick answer ! Unfortunately I have an already existing openldap server that I cannot containerize, my bad I thought it was possible to use this webapp without any modifications to the existing infrastructure.

**Reply by tiredofit** [reply]
Not all is lost however, you can install the appropriate LDIFs and schemas to your existing LDAP server if you'd like. I reverse engineered the install process from official packages [here](https://github.com/tiredofit/docker-openldap-fusiondirectory/blob/fc3b35c9fb6659bf8e1cef724fe50160a325155e/install/assets/custom-scripts/001-install-fusiondirectory.sh#L58).

PS - It's worth the install, a game changer for LDAP Management!

**Reply by Cloud-Kid** [reply]
Thanks for the input, you are doing huge work ! I definitely will look into that, I need to understand the way it works a bit more. I currently use PhpLDAPAdmin but it such a pain in the ***.

I appreciate the time you took !
